x-pack/plugins/cloud/server/util/generate_jwt.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import jwt from 'jsonwebtoken';
+
+export const generateSignedJwt = (userId: string, secret: string): string => {
```

Comment:
NIT/optional: I know it's mostly delegating the logic to `jwt.sign`, but we could still add some unit test coverage for the function.

Reply:
I actually wrote a unit test and it crashed because I hadn't mocked `jsonwebtoken`... then I realized, that's a very tiny unit test.  :-)

I'll add it back.